src/main/java/org/zalando/nakadi/service/subscription/SubscriptionService.java

```diff
@@ -213,6 +217,7 @@ public void deleteSubscription(final String subscriptionId)
         nakadiKpiPublisher.publish(subLogEventType, () -> new JSONObject()
                 .put("subscription_id", subscriptionId)
                 .put("status", "deleted"));
+        subscriptionCache.invalidateSubscription(subscriptionId);
```

Comment:
I believe this call is useless when we have 150 instances

Reply:
but what if you have 1 instance?